integrations/symfony-bundle.rst

```diff
@@ -1,74 +1,60 @@
+    $response = $this->container->get('httplug.client.acme')->sendRequest($request);
 
-\* *These services are always an alias to another service. You can specify your own service or leave the default, which is the same name with `.default` appended. The default services in turn use the service discovery mechanism to provide the best available implementation. You can specify a class for each of the default services to use instead of discovery, as long as those classes can be instantiated without arguments.*
+Installation
+````````````
 
+Install the Httplug bundle with composer and enabled it in your AppKernel.php.
 
-If you need a more custom setup, define the services in your application configuration and specify your service in the `main_alias` section. For example, to add authentication headers, you could define a service that decorates the service `httplug.client.default` with a plugin that injects the authentication headers into the request and configure `httplug.main_alias.client` to the name of your service.
+.. code-block:: bash
 
-.. code-block:: yaml
+    $ composer require php-http/httplug-bundle
 
-    httplug:
-        clients:
-            acme_client: # This is the name of the client
-            factory: 'httplug.factory.guzzle6'
-
-        main_alias:
-            client: httplug.client.default
-            message_factory: httplug.message_factory.default
-            uri_factory: httplug.uri_factory.default
-            stream_factory: httplug.stream_factory.default
-        classes:
-            # uses discovery if not specified
-            client: ~
-            message_factory: ~
-            uri_factory: ~
-            stream_factory: ~
+.. code-block:: php
+
+    public function registerBundles()
+    {
+        $bundles = array(
+            // ...
+            new Http\HttplugBundle\HttplugBundle(),
+        );
+    }
 
+You will find all available configuration at the :doc:`full configuration </integrations/symfony-full-configuration>` page.
 
-Configuration without auto discovery
-````````````````````````````````````
+Web debug toolbar
+`````````````````
+.. image:: assets/img/debug-toolbar.png
+    :align: right
+    :width: 120px
 
-By default we use Puli to auto discover factories. If you do not want to use auto discovery you could use the following configuration (Guzzle):
+When using a client configured with HttplugBundle you will get debug information in the web debug toolbar. It will tell you how many request were made and how many of those that were successful or not. It will also show you detailed information about each request.
+
+Discovery of factory classes
+````````````````````````````
+
+If you want to automatically find our factory classes you should install and enabled ``puli/symfony-bundle``. If you do not want use auto discovery you should specify all the factory classes for you client. The following example show how you configure factory classes using Guzzle.
```

Comment:
be more assertive here:

If you want the bundle to automatically find usable factory classes, install and enable ``puli/symfony-bundle``. If you do not want use auto discovery, you need to specify all the factory classes for you client. The following example show how you configure factory classes using Guzzle: